test/Interop/Cxx/foreign-reference/inheritance.swift

```diff
@@ -30,4 +30,15 @@ TemplatingTestSuite.test("BaseT") {
     assert(bc.isBase)
 }
 
+var FrtInheritanceTestSuite = TestSuite("Foreign references in C++ inheritance")
+
+FrtInheritanceTestSuite.test("ParentChild") {
+  var x = returnValueType()
+  var y = returnRefType()
+  var z = returnDerivedFromRefType()
+  assert(!(type(of: x) is AnyObject.Type), "Expected x to be a value type, but it’s a reference type")
+  assert(type(of: y) is AnyObject.Type, "Expected y to be a reference type, but it’s a value type")
+  assert(type(of: z) is AnyObject.Type, "Expected z to be a reference type, but it’s a value type")
```

Comment:
`assert` isn't the right function to call here. I now see that it's used elsewhere in this test file, which is incorrect. Swift's `assert`s get stripped away in release mode, so this test is effectively ignored when running in release mode. I didn't notice this before.

Could you please use `expectTrue` instead of `assert` in the new test function that you're adding?